### PR TITLE
Add ability to open or close the debug drawer

### LINKED ISSUE #38: Close Drawerlayout nicely
Hey, thanks for this awesome library!

Btw, we ran into the situation where we want to add a feature flag on the `Drawerlayout` and navigate the app to somewhere. The problem we ran into is how to programmatically close the `Drawerlayout` after the transition is completed.

Currently, workaround is kinda hacky like the following;

``` kotlin
//inside the custom class that implements the DebugDrawerModule

activity.findViewById<DrawerLayout>(R.id.debugDrawerId).closeDrawers()
```

Do you have a better way to achieve this?

### PR DESCRIPTION
**Pull request by chris-horner**
Fixes #38

I've run with this solution as I feel its simple, and doesn't give the consumer access to the actual `DrawerLayout` to mess with. Hopefully this works well enough for @kittinunf's use case.

### CODE REVIEW

File: debugdrawer/src/main/kotlin/au/com/gridstone/debugdrawer/DebugDrawer.kt [hunk 1]
```diff
@@ -48,6 +49,22 @@ object DebugDrawer {
    */
   fun with(activity: Activity): Builder = Builder(activity)
 
+  /**
+   * Closes the debug drawer if one has been installed in the activity.
+   */
+  fun closeIn(activity: Activity) {
+    val layout: DrawerLayout? = activity.findViewById(R.id.debugDrawerId)
+    layout?.closeDrawer(GravityCompat.END)
```
Comment (xxfast) [inline debugdrawer/src/main/kotlin/au/com/gridstone/debugdrawer/DebugDrawer.kt]: Not a fan of the silent fail here though

Comment (chris-horner) [reply debugdrawer/src/main/kotlin/au/com/gridstone/debugdrawer/DebugDrawer.kt]: You reckon this method should through an `IllegalStateException`  if you attempt to close the drawer when there isn't one?

Comment (xxfast) [reply debugdrawer/src/main/kotlin/au/com/gridstone/debugdrawer/DebugDrawer.kt]: a throw would be pretty extreme - perhaps a consumable `Boolean` back?

**Comment by chris-horner** [comment]
PR updated 🙂

**Comment by kittinunf** [comment]
Thanks!!

**Pull request merged**